server/views/pages/incident.njk

```diff
@@ -83,7 +83,7 @@
             text: "Reported by"
           },
           {
-            text: report.reportedBy
+            text: nameMappings[report.reportedBy]
```

Comment:
(Nunjacks has the annoying tendency to fail silently)

It's probably a good idea to fallback in case for whatever reason the username was not found.

I know we had in the past shown "System" (or something like that) for system usernames that may not be real usernames.

But I think for now you could keep this simple and just display the original username, something like this could work:

```
text: nameMappings[report.reportedBy] | default(report.reportedBy)
```

although I imagine a plain `or` would also do:

```
text: nameMappings[report.reportedBy] or report.reportedBy
```